### DISCUSSION TITLE
how to add custom style to mapbox?

### DISCUSSION THREAD

**Discussion by Kingflyinger** [apache/superset#18266] [Q&A / Help]
I have searched the way to add custom style to mapbox and found nothing.
but the six existing styles  is too boring, so anyone knows how  to add custom style to mapbox?

**Comment by Kingflyinger** [comment]
is there anyone knows the way? I`m a new learner,but I do feel there is an easy way to add a custom style to the chart.please help me out

**Comment by candy4290** [comment]
I have the save question, how to fix it?help！
